settings/development.py

```diff
@@ -77,3 +77,9 @@
 
 # This will expose all browsable api urls. For dev the default value is true
 API_DEBUG = env.bool('API_DEBUG', default=True)
+
+# MEDIA CONFIGURATION
+# ------------------------------------------------------------------------------
+
+# Media configuration to support deployment of media files while is debug=True or development.
+MEDIA_URL = env("MEDIA_URL", default="/media/")
```

Comment:
This is not needed. Please remove this change.

Reply:
Images will not work without it. Please give it a try.